README.md

```diff
@@ -89,7 +89,21 @@ go get github.com/AllenDang/giu@master
 
 ### Linux
 
-*Need help* here cause I don't have any linux experience.
+First you need to install libraries
+```bash
+# apt install libx11-dev libxcursor-dev libxrandr-dev libxinerama-dev libxi-dev libglx-dev libgl1-mesa-dev libxxf86vm-dev
```

Comment:
Hi!
here are dependencies for RedHat-based OS:
`sudo dnf install -y libX11-devel libXcursor-devel libXrandr-devel libXinerama-devel mesa-libGL-devel libXi-devel`

Reply:
Okay! Is there anything special for cross-compiling?